native/dlahqr.go

```diff
@@ -14,7 +14,7 @@ import (
 // upper Hessenberg matrix H, using the double-shift/single-shift QR algorithm.
 //
 // h and ldh represent the matrix H. Its contents on return depends on the value
-// of info. Dlahqr works primarily with the Hessenberg submatrix
+// of ok. Dlahqr works primarily with the Hessenberg submatrix
```

Comment:
Great suggestion, thanks. You are right, everything is actually ok even if `ok` is false. Therefore `converged` is a much better name. 

I renamed the return to `converged` and slightly reordered the paragraphs about `converged`.